app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFieldsIdRepository.scala

```diff
@@ -85,14 +87,20 @@ class SubscriptionFieldsIdMongoRepository @Inject() (mongoDbProvider: MongoDbPro
   }
 
   override def fetchById(id: String): Future[Option[SubscriptionFields]] = {
-    collection.find(Json.obj("id" -> id)).one[SubscriptionFields]
+    val selector = Json.obj("id" -> id)
```

Comment:
`Json.obj("id" -> id)` is used in two methods here.
Why don't you extract it in a constant or private method?
Let's make the code reusable.
Let's avoid repetition of code.

Reply:
done